.gitlab-ci.yml

```
@@ -32,8 +31,48 @@ unit_tests_julia1.9:
   tags:
     - cpuonly
 
-verify-unit-test-deps_julia1.9:
-  image: julia:1.9
+unit_tests_releases:
+  extends: .untit_test_template
+  parallel:
+    matrix:
+      - JULIA_VERSION: ["1.6", "1.7", "1.8", "1.9", "1.10"]
```

```suggestion
      - JULIA_VERSION: ["1.6", "1.7", "1.8", "1.9", "1.10", "rc"]
```
We added the release candidate in the other packages too, I think we should have it here too.